{"name": "test_invalid_login", "status": "broken", "statusDetails": {"message": "RuntimeError: Username field not found (tried multiple locators)", "trace": "driver = <appium.webdriver.webdriver.WebDriver (session=\"572412b8e8d34caf2b71c8b8aae9c22be44d7735\")>\n\n    @pytest.mark.smoke\n    def test_invalid_login(driver):\n        lp = LoginPage(driver)\n        # locked_out_user is a known invalid account in the sample app\n>       lp.login(\"locked_out_user\", \"secret_sauce\")\n\ntests\\test_invalid_login.py:8: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npages\\login_page.py:69: in login\n    self.enter_username(username)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <pages.login_page.LoginPage object at 0x00000214321B2E90>, username = 'locked_out_user'\n\n    def enter_username(self, username):\n        loc = self._find_first(self._locator_username_candidates(), timeout=6)\n        if not loc:\n>           raise RuntimeError(\"Username field not found (tried multiple locators)\")\nE           RuntimeError: Username field not found (tried multiple locators)\n\npages\\login_page.py:49: RuntimeError"}, "attachments": [{"name": "screenshot-android", "source": "f98e1183-a225-49cb-88c5-f173b660fba9-attachment.png", "type": "image/png"}], "start": 1757853194868, "stop": 1757853227149, "uuid": "f9e8d389-6ae3-45ea-9017-af499c75b051", "historyId": "55f35891554f4a3a55efb423475adf20", "testCaseId": "55f35891554f4a3a55efb423475adf20", "fullName": "tests.test_invalid_login#test_invalid_login", "labels": [{"name": "tag", "value": "smoke"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_invalid_login"}, {"name": "host", "value": "DB-L-174"}, {"name": "thread", "value": "1268-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_invalid_login"}], "titlePath": ["tests", "test_invalid_login.py"]}